lightning_pose/utils/scripts.py

```diff
@@ -152,11 +177,14 @@ def get_data_module(
         )
     else:
         if cfg.model.model_type == "heatmap_mhcrnn" and cfg.dali.context.train.batch_size < 5:
-            raise ValueError(
+            raise ValidationError(
```

Comment:
@themattinthehatt I changed your ValueError to ValidationError for consistency. My only reason for preferring validation error is it looks more like a user error to the end user.